src/wasm_offset_converter.js

```diff
@@ -88,7 +88,7 @@ function WasmOffsetConverter(wasmBytes, wasmModule) {
               unsignedLEB128(); // skip function type
               break;
             case 1: // table import
-              ++offset; // FIXME: should be SLEB128
+              ++offset; // skip elem type
```

Comment:
Drive-by fix: I think this should be `uint8`, because this is `elem_type`: https://webassembly.github.io/spec/core/syntax/types.html#syntax-tabletype

Reply:
I think in the MVP's design it was indeed one byte, but only because it had a single possible value. With typed function references and GC we could have an arbitrary user type here, so I think it does need to be a LEB.

Reply:
Will change to `unsignedLEB128()` then.